cmd/go-getter/go.mod

```diff
@@ -0,0 +1,13 @@
+module github.com/go-getter/cmd/go-getter/v2
+
+go 1.14
+
+replace github.com/hashicorp/go-getter/v2 => ../..
```

Comment:
Question: Is this line necessary ? What does it do/change ?

Reply:
Ah nevermind, I got it, it allows to use the latest go-getter/v2 without having to update the go.sum for each release; nice !